lib/charms/archive_auth_mirror/utils.py

```diff
@@ -52,8 +52,12 @@ def get_virtualhost_config(hookenv=hookenv):
         'basic_auth_file': str(paths['basic-auth'])}
 
 
-def install_resources(base_dir=None):
+def install_resources(base_dir=None, cron_dir=None):
```

Comment:
It's probably a too big change to do now, but instead of having base_dir and cron_dir, it would be better to have root_dir, which would default to '/'. That way get_paths() would also accept root_dir and return the absolute paths of everything.